2.1.1/buster/build-time-pip-constraints.txt

```diff
@@ -0,0 +1,245 @@
+# Constraint the version pip will install during docker build
+
+adal==1.2.7
+alembic==1.6.2
+amqp==2.6.1
+apache-airflow-providers-amazon==1.4.0
+apache-airflow-providers-celery==1.0.1
+apache-airflow-providers-cncf-kubernetes==1!1.2.1
+apache-airflow-providers-elasticsearch==1.0.4
+apache-airflow-providers-ftp==1.1.0
+apache-airflow-providers-google==3.0.0
+apache-airflow-providers-imap==1.0.1
+apache-airflow-providers-microsoft-azure==2.0.0
+apache-airflow-providers-mysql==1.1.0
+apache-airflow-providers-postgres==1.0.2
+apache-airflow-providers-redis==1.0.1
+apache-airflow-providers-slack==3.0.0
+apache-airflow-providers-sqlite==1.0.2
+apache-airflow-providers-ssh==1.3.0
+apispec==3.3.2
```

Comment:
Does this come from https://github.com/apache/airflow/blob/constraints-2-1/constraints-3.7.txt ?

Reply:
can i replace the entire contents of it here with 2-1contraints?

Reply:
@kaxil

Reply:
Yeah

Reply:
Done